applications/compress/app.json

```diff
@@ -49,9 +49,9 @@
                 },
                 {
                     "name": "Archive File Name",
-                    "description": "Output archive file name. The selected Compression Type will be appended to this filename.",
+                    "description": "Output archive file name. The selected Compression Type will be appended to this filename. Keep the file name inside single quotes. Any special characters in the file name will be replaced with an underscore.",
                     "inputMode": "REQUIRED",
-                    "arg": "Archive"
+                    "arg": "'Archive'"
```

Comment:
Does this need to be double wrapped in quotes?

Reply:
Hi Sal, so I put the double quotes for making the argument string and since the single quotes are inside double quotes, single quotes would appear in the input field. I put the single string there so that if any user wants to use space or other characters, they could put them inside the quotes. I also added description accordingly. The single quotes are for easy-understanding.

Reply:
<img width="762" alt="image" src="https://github.com/TACC/WMA-Tapis-Templates/assets/43958517/37396f98-98d2-4322-b479-ac8678e6f3b0">